net.tedomum.CapBattleship.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "net.tedomum.CapBattleship",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "capbattleship",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "cleanup": [],
+    "modules": [
+        {
+            "name": "python3-pygame",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app ."
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.pythonhosted.org/packages/bc/0c/d164411381bada8feae57359aa9296defbed3035f4b55bef6a271cb0573c/pygame-2.0.0.tar.gz",
+                    "sha256": "63b038da116a643046181b02173fd894d87d2f85ecfd6aa7d5ece73c6ef501e9"
+                },
+                {
+                    "type": "patch",
+                    "path": "disable-portmidi.patch"
+                }
+            ]
+        },
+        {
+            "name": "capbattleship",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app .",
+                "mkdir -p /app/share/icons/hicolor/256x256/apps/",
+                "mkdir -p /app/share/applications/",
+                "install -Dm644 net.tedomum.CapBattleship.metainfo.xml /app/share/metainfo/net.tedomum.CapBattleship.metainfo.xml",
+                "mv /app/share/pixmaps/capbattleship.png /app/share/icons/hicolor/256x256/apps/net.tedomum.CapBattleship.png",
+                "sed '/^Icon/s/=.*/=net.tedomum.CapBattleship/' /app/share/applications/capbattleship.desktop > /app/share/applications/net.tedomum.CapBattleship.desktop",
```

Comment:
Should be renamed upstream otherwise use desktop-file-edit

Reply:
This is already fixed upstream, I’m just waiting for a release from them.